src/index.md

```diff
@@ -26,7 +26,7 @@ In many GitHub projects the standard way to propose a change is to create a pull
 
 ### 1. Sign the OCA
 
-Oracle are the stewards of the OpenJDK project, and owns the Java brand. In order to make your patch available for review you must first sign the [Oracle Contributor Agreement](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html) (OCA). This agreement gives Oracle and you as a contributor joint copyright interests in the code. You will retain your copyright while also granting those rights to Oracle.
+Oracle is the stewards of the OpenJDK project, and owns the Java brand. In order to make your patch available for review you must first sign the [Oracle Contributor Agreement](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html) (OCA). This agreement gives Oracle and you as a contributor joint copyright interests in the code. You will retain your copyright while also granting those rights to Oracle.
```

Comment:
OK, now it's American/British usage (whether a company name is singular or plural) vs. noun-verb agreement. I thought "Oracle are the stewards..." was ok for British usage (but as an American, what do I know). I'm also OK with making "Oracle" singular but then "steward" should be singular too. That is, "Oracle is the steward of the OpenJDK project, ...."

Reply:
Fixed.